extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-failover.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "powershell",
+            "display_name": "PowerShell",
+            "language": "powershell"
+        },
+        "language_info": {
+            "name": "powershell",
+            "codemirror_mode": "shell",
+            "mimetype": "text/x-sh",
+            "file_extension": ".ps1"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Create and configure Failover Cluster\n",
+                "\n",
+                "## Description\n",
+                "\n",
+                "Notebook to walk through creating a failover cluster on on-premises SQL Server Database."
+            ],
+            "metadata": {
+                "azdata_cell_guid": "6af59d69-ade7-480a-b33e-52a86fe5bfd3"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Note: before running the steps below, make sure you have the following:\n",
+                "\n",
+                "- A resource group with a domain controller.\n",
+                "- One or more domain joined VMs in Azure running SQL Server 2016 (or later) in the _same_ availability set or _different_ availability zones that have been registered with the SQL IaaS Agent extension\n",
+                "- Domain joined VMs must have either a \"Enterprise\" or \"Developer\" image SKU to create a cluster in this notebook\n",
+                "\n",
+                "Tutorial on how to set up a domain with VMs can be found [here.](https://docs.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/availability-group-manually-configure-prerequisites-tutorial)\n",
+                "\n",
+                "Steps of this procedure include:\n",
+                "\n",
+                "1. Connect to Azure subscription\n",
+                "2. Provision resource group for SQL VM migration\n",
+                "3. Create a storage account\n",
+                "4. Define cluster metadata\n",
+                "5. Add VMs to the cluster\n",
+                "6. Validate cluster"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "b911ea4f-a3d8-4ac1-bff6-6c5eb1b514f9"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#Set Parameters\r\n",
+                "$ClusterName = \"\"                        # The name of the SQL VM Cluster.\r\n",
```

Comment:
I like to start each notebook with $Subscription, $ResourceGroupName at least. Those are the most common used. The more generally used go to the top. The specific ones for this notebook towards the bottom.